src/mpl_widget.ts

```diff
@@ -275,17 +287,49 @@ export class MPLCanvasModel extends DOMWidgetModel {
         this.image = new Image();
 
         this.image.onload = () => {
+            // In case of an embedded widget, the initial size is not correct
+            // and we are not receiving any resize event from the server
+            if (this.disabled) {
```

Comment:
This is the second important change in this PR.

Here, we need a special case for when the widget is disabled (no counterpart in the kernel, or no kernel at all), where we display the image that is part of the model.

Reply:
Am I right that this will come up in many situations, not just nbconvert, right?